### ISSUE TITLE
Find all similar images in database

### ISSUE THREAD

**Issue by JannikZed** [pavlovml/match#27]
As far as I understand the system, you could only compare a picture send in via URL or formed data with the ones already in place. How could we compare all pictures with each others in the database to find all that are similar?

**Comment by dgtlmoon** [comment]
Hey, yes I see the problem here, is that the project is kind of vague how it describes itself, you can only compare images and not search... which would mean N* searches for all images... ouch

**Comment by dgtlmoon** [comment]
But, back to the original project... https://image-match.readthedocs.io/en/latest/searches.html   ?

**Comment by lostpebble** [comment]
Yes, this something I would really love to have. Right now the only way to search for similar images is to upload one? (be it by the url passed, or an actual image sent with POST)

Is there no way for us to just pass a `filepath` already in our database and get similar results from that?
